blockdevice/filesystem/msdos/superblock.go

```diff
@@ -0,0 +1,47 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+package msdos
+
+import (
+	"bytes"
+)
+
+const (
+	// Magic12 is the FAT12 magic signature.
+	Magic12 = "FAT12"
+	// Magic16 is the FAT16 magic signature.
+	Magic16 = "FAT16"
+)
+
+// SuperBlock represents the vfat super block.
+type SuperBlock struct {
+	_     [0x2b]uint8
+	Label [11]uint8
+	Magic [8]uint8
+	_     [0x1cd]uint8
```

Comment:
should we add a comment to link to some docs where these values are defined?

Reply:
yes, I can, it was Wikipedia :)